resolwe_bio/processes/alignment/bowtie.yml

```diff
@@ -231,8 +231,10 @@
   name: Bowtie 2.2.3
```

Comment:
Please drop the Bowtie version from the process name. Bowtie might get updated in the docker image, but the process name will stay the same.